src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -36,17 +63,35 @@ class PhEDExInjectorPoller(BaseWorkerThread):
 
     Poll the DBSBuffer database and inject files as they are created.
     """
-    def __init__(self, config, phedex, nodeMappings):
+    def __init__(self, config):
         """
         ___init___
 
         Initialise class members
         """
         BaseWorkerThread.__init__(self)
-        self.config = config
-        self.phedex = phedex
         self.dbsUrl = config.DBSInterface.globalDBSUrl
 
+        self.pollCounter = 0
+        self.subFrequency = None
```

Comment:
Because the default is to skip the second polling cycle. Using None to indicate that avoids having to use a second parameter.